src/coreclr/vm/precode_portable.hpp

```diff
@@ -31,6 +34,32 @@ class PortableEntryPoint final
 
 public:
     void Init(MethodDesc* pMD);
+
+    // Query methods for entry point state.
+    bool IsValid() const;
+
+    bool IsByteCodeCompiled() const
```

Comment:
This name makes is sound like that the byte code was compiled into something that's not right. The byte code is the product of the compilation.

I would call this `HasInterpreterCode`. I think we use "Interpreter Code" in this context in majority of the codebase (e.g. `MethodDesc::GetInterpreterCode`).